projects/igniteui-angular/src/lib/grids/hierarchical-grid/row-island.component.ts

```diff
@@ -138,6 +138,20 @@ export class IgxRowIslandComponent extends IgxHierarchicalGridBaseDirective
     @Output()
     public onGridCreated = new EventEmitter<IGridCreatedEventArgs>();
 
+    /**
```

Comment:
Description and code snippet should be updated to match what is added in the ReadMe.